### PR TITLE
fix(titles): A lot of layout title fixes

### PR DESCRIPTION
**Pull request by SpeedyD**
I present to you: A TON of title fixes, consistency edits and replacement of old `@stop` Laravel code.

Exceptions include:
- account/banned.blade.php (I figured when banned, the title Banned is straight forward, should not need Account in front)
- account/deactivated.blade.php (I figured when deactivated, the title Deactviated is straight forward, should not need Account in front)

As well as the following:
- admin/submissions/submission.blade.php
- home/submissions.blade.php
- home/bug_report_index.blade.php
- home/report.blade.php

Of which all require quite a bit more edits than just this, and will be done in a seperate PR.

### CODE REVIEW

File: resources/views/character/design/features.blade.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @extends('character.design.layout')
 
 @section('design-title')
-    Design Approval Request (#{{ $request->id }}) :: Comments
+    Request (#{{ $request->id }}) :: Comments
```
Comment (itinerare) [inline resources/views/character/design/features.blade.php:4]: Traits, not comments!

File: resources/views/character/design/index.blade.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @extends('character.design.layout')
 
 @section('design-title')
-    Design Approvals
+    Index
```
Comment (AW0005) [inline resources/views/character/design/index.blade.php:4]: Maybe "Design Submission" might be a little more descriptive?

Comment (AW0005) [reply resources/views/character/design/index.blade.php:4]: Oh nvm, I scrolled down just a little bit more and see where you added the prefix my bad

**Review by itinerare — CHANGES_REQUESTED** [review]

**Review by AW0005 — APPROVED** [review]

File: resources/views/character/design/features.blade.php (hunk 1 above)
Comment (SpeedyD) [reply resources/views/character/design/features.blade.php:4]: I KNEW I missed one! Hang on.

**Review by itinerare — APPROVED** [review]

**Pull request merged**